ingestclient/core/engine.py

```diff
@@ -340,12 +340,22 @@ def run(self):
                                                  key_parts["t_index"])
 
             try:
+
                 metadata = {'chunk_key': msg['chunk_key'],
                             'ingest_job': self.ingest_job_id,
-                            'parameters': self.job_params,
+                            'parameters': dict(self.job_params),
                             'x_size': self.config.config_data['ingest_job']["tile_size"]["x"],
                             'y_size': self.config.config_data['ingest_job']["tile_size"]["y"],
                             }
+                # The following parameters are not needed during the ingest job
+                # however SPDB will fail to load correctly without the keys.
+                # The values are removed as there can only be 2K of metadata.
+                # If we need more room we could delete these keys and automatically add them again
+                # before instantiating SPDB.
+                metadata['parameters']['OBJECTIO_CONFIG']['page_in_lambda_function'] = ""
```

Comment:
We could also blank these keys out on the endpoint side so the client doesn't need to _know_ as much about the internals.